### ISSUE TITLE
Gallery tests are always failing on CI

### ISSUE THREAD

**Issue by t-b** [NeurodataWithoutBorders/pynwb#969]
They pass here locally with b94a3238 (Merge pull request #963 from NeurodataWithoutBorders/update_authors, 2019-05-29) and 1.0.2 of hdmf.

**Comment by oruebel** [comment]
The errors occurs in the allensdk with the import scipy.misc.imresize which has been removed in scipy https://docs.scipy.org/doc/scipy/reference/release.1.3.0.html?highlight=imresize#scipy-interpolate-changes This is being addressed in the AllenSDK I believe in  https://github.com/AllenInstitute/AllenSDK/issues/660 . Looking at the AllenSDK releases, it looks like they released 0.16.3 recently which fixes the issue. It may be as simple as updating the version of our dependency from 0.16.0 to 0.16.3 here https://github.com/NeurodataWithoutBorders/pynwb/blob/08e7f7ace648411523db4881a5181e060d7312d4/requirements-doc.txt#L6


Here the error that occurs in the gallery CI
```
2019-05-30 18:45:34,627 - INFO - Executing docs/gallery/domain/brain_observatory.py
Traceback (most recent call last):
  File "test.py", line 85, in run_example_tests
    _import_from_file(script)
  File "test.py", line 64, in _import_from_file
    return imp.load_source(os.path.basename(script), script)
  File "/home/circleci/project/.tox/gallery-py37/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "docs/gallery/domain/brain_observatory.py", line 20, in <module>
    from allensdk.core.brain_observatory_cache import BrainObservatoryCache
  File "/home/circleci/project/.tox/gallery-py37/lib/python3.7/site-packages/allensdk/core/brain_observatory_cache.py", line 39, in <module>
    from allensdk.api.queries.brain_observatory_api import BrainObservatoryApi
  File "/home/circleci/project/.tox/gallery-py37/lib/python3.7/site-packages/allensdk/api/queries/brain_observatory_api.py", line 43, in <module>
    import allensdk.brain_observatory.stimulus_info as stimulus_info
  File "/home/circleci/project/.tox/gallery-py37/lib/python3.7/site-packages/allensdk/brain_observatory/stimulus_info.py", line 39, in <module>
    from scipy.misc import imresize
ImportError: cannot import name 'imresize' from 'scipy.misc' (/home/circleci/project/.tox/gallery-py37/lib/python3.7/site-packages/scipy/misc/__init__.py)
```

**Comment by oruebel** [comment]
I just made a pull request to update the AllenSDK version. Let's see if that fixes the issue.
